From private repo

v-alje opened the pull request.

azurecla commented:
Hi __@baywet__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

tysonn commented:
Thanks for this contribution; the team reviewed and agreed to accept. We appreciate it!

dsk-2015 committed:
.Net/.Net direct methods - new article

dsk-2015 committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into dsk_altiot

dsk-2015 committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into dsk_altiot

genlin committed:
add new article

genlin committed:
correct typo

genlin committed:
change the file name

genlin committed:
correct typo

genlin committed:
correct typo

genlin committed:
add new content

genlin committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into 53201

genlin committed:
add more information

genlin committed:
correc typo

genlin committed:
correct typo

genlin committed:
fix the issiue

genlin committed:
fix format issue

genlin committed:
fix format issue

genlin committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into 53201

genlin committed:
update

genlin committed:
remove article

genlin committed:
more info

genlin committed:
add info

dsk-2015 committed:
intermediate

dsk-2015 committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into dsk_altiot

genlin committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into 53201

dsk-2015 committed:
csharp direct methods

dsk-2015 committed:
toc

genlin committed:
correct typo

genlin committed:
update the picture

dsk-2015 committed:
fix

dsk-2015 committed:
fix2

genlin committed:
correct link issue

dsk-2015 committed:
dev identity include

dsk-2015 committed:
borders to pngs

dsk-2015 committed:
fix link

genlin committed:
resize image

EdwardDing committed:
Add GitHub, LucidChart, Samanage, ThousandEyes and Zendesk provisioning docs

Commit:
application gateway - url redirects

Commit:
updates to the docs

Commit:
updates to article

Commit:
redirect article

EdwardDing committed:
Modified the error style ,grammar and spelling

EdwardDing committed:
improve the style

EdwardDing committed:
improve the style according to the comments

EdwardDing committed:
put the period inside the quotes

genlin committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into 53201

Commit:
updates

Commit:
updates to articles

Commit:
corrected links

genlin committed:
merge the content

genlin committed:
add more info

genlin committed:
correct typo

genlin committed:
change wording

genlin committed:
remove files

genlin committed:
redirect the article

EdwardDing committed:
improve format as comments said

EdwardDing committed:
Add border for all the pictures

genlin committed:
remove the images about classic storage

Lichard committed:
Fix rules engine TOC

genlin committed:
remove image folder

genlin committed:
remove classic reference from other article

iamclement committed:
AzME, update iOS get started

iamclement committed:
AzME, update background location doc for iOS 11

iamclement committed:
AzME, release iOS 4.1.0

spboyer committed:
Clean up and clarity changes

#sign-off

iamclement committed:
AzME, remove iOS 6 methods

Commit:
updates to overview

Commit:
added images and updated faq

Commit:
updates to the faq article

Commit:
Update data-lake-analytics-manage-use-powershell.md

Commit:
Update data-lake-analytics-manage-use-powershell.md

davidmu1 committed:
freshness updates

kgremban committed:
RBAC freshness review

dsk-2015 committed:
Removed Node.js left-overs in the headers

dsk-2015 committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into dsk_altiot

kgremban committed:
two more

davidmu1 committed:
corrections

RicksterCDN committed:
updating ToC to properly link to Oracle workload

Commit:
Merge pull request #17621 from saveenr/patch-7

Update data-lake-analytics-manage-use-powershell.md

Juliako committed:
Update media-services-static-packaging.md

Juliako committed:
Update media-services-autogen-bitrate-ladder-with-mes.md

Commit:
Merge pull request #17627 from Juliako/patch-8

Update media-services-static-packaging.md

Juliako committed:
Update media-services-autogen-bitrate-ladder-with-mes.md

Juliako committed:
Update media-services-custom-mes-presets-with-dotnet.md

Juliako committed:
Update media-services-custom-mes-presets-with-dotnet.md

GitHubber17 committed:
Merge pull request #14370 from dsk-2015/dsk_altiot

CSharp Csharp Direct Methods new article

Juliako committed:
Update media-services-deliver-keys-and-licenses.md

Juliako committed:
Update media-services-dotnet-long-operations.md

v-thepet committed:
Merge pull request #17626 from RicksterCDN/ToCFixes

updating ToC to properly link to Oracle workload

TomShinder committed:
Update azure-operational-security.md

fixed typo.

GitHubber17 committed:
Merge pull request #17624 from kgremban/jul18_freshnessrbac

RBAC freshness review

Commit:
Merge pull request #17633 from TomShinder/patch-18

Update azure-operational-security.md

Commit:
Update data-lake-analytics-manage-use-powershell.md

MandiOhlinger committed:
updated service to be biztalk-services, instead of logic-apps

v-thepet committed:
tonight's broken links

Commit:
Merge pull request #17634 from saveenr/patch-1

Update data-lake-analytics-manage-use-powershell.md

Commit:
updates based on feedback:

Commit:
Merge pull request #17635 from v-thepet/roadmap2

production bug: tonight's broken links

Juliako committed:
Update media-services-dotnet-telemetry.md

Juliako committed:
Update media-services-autogen-bitrate-ladder-with-mes.md

v-stgreg committed:
Added text that TaskWebApp needs to be the startup project

Juliako committed:
Update media-services-deliver-keys-and-licenses.md

Commit:
Merge pull request #17636 from MandiOhlinger/master

updated service to be biztalk-services, instead of logic-apps

Commit:
Update data-lake-analytics-manage-use-powershell.md

EdwardDing committed:
make the screenshots intented

Commit:
Merge pull request #17638 from saveenr/patch-2

Update data-lake-analytics-manage-use-powershell.md

Commit:
Merge pull request #17639 from v-stgreg/active-directory-b2c

Added text that TaskWebApp needs to be the startup project

GitHubber17 committed:
Merge pull request #17030 from EdwardDing0302/private/v-fedin/ProvisionDoc

Add GitHub, LucidChart, Samanage, ThousandEyes and Zendesk provisioni…

GitHubber17 committed:
Merge pull request #17119 from georgewallace/appgatewayredirect

app gateway : redirect

iamclement committed:
AzME iOS fixed release note

iamclement committed:
AzME iOS fix overview

neilpeterson committed:
remove cloud-shell

GitHubber17 committed:
Merge pull request #17640 from clpolet/master

AzME iOS SDK 4.1.0

GitHubber17 committed:
Merge pull request #17622 from davidmu1/templateupdate071817

freshness updates

GitHubber17 committed:
Merge pull request #17528 from Lichard/patch-2

Fix rules engine TOC

GitHubber17 committed:
Merge pull request #17629 from Juliako/patch-8

Update media-services-custom-mes-presets-with-dotnet.md

GitHubber17 committed:
Merge pull request #17630 from Juliako/patch-10

Update media-services-deliver-keys-and-licenses.md

GitHubber17 committed:
Merge pull request #17632 from Juliako/patch-11

Update media-services-dotnet-long-operations.md

GitHubber17 committed:
Merge pull request #17637 from Juliako/patch-12

Update media-services-dotnet-telemetry.md

GitHubber17 committed:
Merge pull request #17628 from Juliako/patch-9

Update media-services-autogen-bitrate-ladder-with-mes.md

Juliako committed:
Update media-services-encode-with-premium-workflow.md

Commit:
Update data-lake-analytics-data-lake-tools-get-started.md

Juliako committed:
Update media-services-dotnet-encode-with-media-encoder-standard.md

Juliako committed:
Update media-services-encode-with-premium-workflow.md

Commit:
Merge pull request #17643 from saveenr/patch-3

Update data-lake-analytics-data-lake-tools-get-started.md

Juliako committed:
Update media-services-face-and-emotion-detection.md

Juliako committed:
Update media-services-dotnet-encode-with-media-encoder-standard.md

Juliako committed:
Update media-services-face-and-emotion-detection.md

Juliako committed:
Update media-services-face-and-emotion-detection.md

Juliako committed:
Update media-services-generate-fmp4-chunks.md

Juliako committed:
Update media-services-face-and-emotion-detection.md

Juliako committed:
Update media-services-dotnet-encode-with-media-encoder-standard.md

Juliako committed:
Update media-services-motion-detection.md

Juliako committed:
Update media-services-process-content-with-indexer2.md

Juliako committed:
Update media-services-face-and-emotion-detection.md

Juliako committed:
Update media-services-video-optical-character-recognition.md

Juliako committed:
Update media-services-video-summarization.md

genlin committed:
remove selector file

Juliako committed:
Update media-services-video-optical-character-recognition.md

Commit:
Merge pull request #17652 from Juliako/patch-16

Update media-services-video-summarization.md

Commit:
Merge pull request #17641 from neilpeterson/dcos-remove-cloudshell

remove cloud-shell

v-jotel committed:
P: Potencial Typos on Service Names

-Streaming Analytics was replaced by Stream Analytics
-Azure IoT Hubs was re placed by Azure IoT Hub

Commit:
Merge pull request #17661 from v-jotel/patch-21

P: Potencial Typos on Service Names

Commit:
remove "send utterance to all apps"

Commit:
Merge pull request #17662 from DeniseMak/faq-bugfix-7-19

remove "send utterance to all apps"

itechedit committed:
Merge pull request #16533 from genlin/53201

storage-use-storage-explorer-to-delete-object.md

itechedit committed:
Merge pull request #17642 from Juliako/patch-8

Update media-services-encode-with-premium-workflow.md

Commit:
Merge pull request #17566 from spboyer/patch-4

Clean up and clarity changes

Commit:
Added link to specific permissions

Commit:
Update virtual-network-peering-overview.md

itechedit committed:
Merge pull request #17644 from Juliako/patch-9

Update media-services-dotnet-encode-with-media-encoder-standard.md

v-shils committed:
Merge pull request #17645 from Juliako/patch-10

Update media-services-face-and-emotion-detection.md

Commit:
Merge pull request #17664 from jimdial/patch-93

Added link to specific permissions

v-shils committed:
Merge pull request #17647 from Juliako/patch-12

Update media-services-generate-fmp4-chunks.md

iainfoulds committed:
Fixing metadata

v-shils committed:
Merge pull request #17648 from Juliako/patch-13

Update media-services-motion-detection.md

Commit:
Merge pull request #17665 from iainfoulds/vmmetadatafixes

VM metadata fixes

v-shils committed:
Merge pull request #17649 from Juliako/patch-14

Update media-services-process-content-with-indexer2.md

v-shils committed:
Merge pull request #17651 from Juliako/patch-15

Update media-services-video-optical-character-recognition.md

v-alje committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs into FromPublicRepo

v-alje committed:
Merge pull request #17667 from MicrosoftDocs/FromPublicRepo

From public repo
